core/src/main/java/com/scalar/db/common/CheckedDistributedStorageAdmin.java

```diff
@@ -320,6 +320,15 @@ public void addRawColumnToTable(
     }
   }
 
+  @Override
+  public Set<String> getNamespaceNames() throws ExecutionException {
+    try {
+      return admin.getNamespaceNames();
+    } catch (ExecutionException e) {
+      throw new ExecutionException(CoreError.GETTING_NAMESPACE_NAMES_FAILED.buildMessage(), e);
+    }
+  }
```

Comment:
This differs from the implementation of ScalarDB v4  where the system namespace is added on the `CheckedDistributedStorageAdmin` layer. 
https://github.com/scalar-labs/scalardb/blob/85c9bd388a582150bc9b0b70f303617f6dd58db5/core/src/main/java/com/scalar/db/common/CheckedDistributedStorageAdmin.java#L296-L310

It's because, on ScalarDB v3, each storage admin has its own configuration to set the system namespace. So, each storage admin will add the system namespace to the list of user namespaces.